CHANGELOG.md

```diff
@@ -69,6 +69,9 @@ Main (unreleased)
 - Fix an issue with static integrations-next marshaling where non singletons
   would cause `/-/config` to fail to marshal. (@erikbaranowski)
 
+- Fix an issue where agent logs are emitted before correctly determine 
+  logging format. (@hainenber)
```

Comment:
Fix an issue where agent logs are emitted before the logging format is correctly determined.

Reply:
I've addressed. PTAL